mapboxgl/viz.py

```diff
@@ -39,18 +39,13 @@ def __init__(self,
         """
         if access_token is None:
             access_token = os.environ.get('MAPBOX_ACCESS_TOKEN', '')
-        if not access_token.startswith('pk'):
-            raise TokenError('Mapbox access token must be public (pk). ' \
-                             'Please sign up at https://www.mapbox.com/signup/ to get a public token. ' \
-                             'If you already have an account, you can retreive your token at https://www.mapbox.com/account/.')
```

Comment:
I would advise strongly against removing this check. Allowing secret keys to pass through unchecked is a serious security vulnerability. Since notebooks will most likely shared on github etc., we run the significant risk of accidentally exposing a user's credentials.

Reply:
In that case I would suggest raising an error if the token starts with "sk", this way the users can still use the maps without a mapbox token, and the js library will take care of error handling if they try to use a mapbox resource, but we still ensure they won't publish secrets.

Reply:
I'm good with throwing an error if the token begins with `sk`.